vscode/src/test/suite/general/extension.test.ts

```diff
@@ -166,7 +166,7 @@ suite('Extension Test Suite', function () {
     if (refactorActions && refactorActions.length > 0) {
       for await (const action of refactorActions) {
         if (action.command && action.command.arguments) {
-          if (action.command.command === myExtension.COMMAND_PREFIX + ".surround.with") {
+          if (action.command.command === extConstants.COMMAND_PREFIX + ".surround.with") {
```

Comment:
Should we replace the command names with the references from *commands.ts*?

Reply:
Done